homework1/task5_distance.py

```diff
@@ -0,0 +1,16 @@
+"""Определение расстояния в 2D пространстве между двумя точками, по координатам поданными на вход"""
+
+#стараюсь соблюдать PEP8 и ограничивать строку 80 символами
+#поэтому в следующей строке добавился print() перед input()
+print('Введите координаты точки А через пробел',end=" ")
+point_A = list(map(float,input().split(sep=' ')))
```

Comment:
```suggestion
x1, y1 = map(float, input().split())
```
Можно использовать групповое присваивание и не нужно потом обращаться по индексам.

Reply:
> Можно использовать групповое присваивание и не нужно потом обращаться по индексам.
А если условия изменятся, и точек будет больше чем две и пространство будет трех и более мерным?
Поэтому и возникла идея обозначения координат через список. Доработать код под возможные изменения было бы проще. Или я не прав в своих соображениях?

Reply:
В любом случае код придется дорабатывать, если вам необходимо считать расстояние между точками во многомерном пространстве.